src/js/common.js

```diff
@@ -153,12 +162,43 @@ module.exports.loadLatestAssets = (variant, openjdkImp, releaseType, release, ty
   queryAPI(release, url, openjdkImp, type, errorHandler, handleResponse);
 }
 
+module.exports.getChecksum = url => checksums[url]
+
+module.exports.loadChecksum = (url, message) => {
+  return new Promise((resolve, reject) => {
+    loadUrl(url, response => {
+      try {
+        const reader = new FileReader();
+        // This fires after the blob has been read/loaded.
+        reader.addEventListener('loadend', e => {
+          // todo@userquin: fix this -> https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement
+          console.info(e.srcElement);
```

Comment:
Would need to remove the console logging when done

Reply:
yes, once tested this console will be removed

Reply:
In fact, I must check if responseText from xhr redirect response just returns the checksum line: in that case we just revert all changes made to handle xhr response on `commons.js` and here just remove the FileReader code...

Reply:
there is no need to use FileReader, just xhr.responseText works

@karianna to listen for events on html elements you add event listeners to them, provinding the event in question and a callback function: for example, a button can emit multiple events such as click, then we register an event listener to be notified when the user clicks on it:
```javascript
someButton.addEventListener("click", function(e) {
  // handle the event
}
```
The callback function can receive the event that fires the callback, you can call what the name you want. Using arrow functions, we can reduce the code removing extra code that is implicit:
```javascript
someButton.addEventListener("click", e => {
  // handle the event
}
```